webrtc-stats.html

```diff
@@ -2777,6 +2790,17 @@ <h2>
                   frame has been produced this attribute is missing.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>bitDepth</code></dfn> of type <span class="idlMemberType">unsigned
+                long</span>
+              </dt>
+              <dd>
+                <p>
+                  The bit depth per pixel of the last frame originating from this source. Before a
+                  frame has been produced this attribute is missing.
```

Comment:
Is this enough or do we also want frameBitDepth on the RTCOutboundRtpStreamStats representing the bit depth *after* encoding? If the bit depth is higher from the camera than what we encode.

Reply:
You're right, that could happen. Added it back.
Thanks for the comment!